apps/fluent-tester/src/TestComponents/Common/MenuPicker.tsx

```diff
@@ -1,43 +1,11 @@
-import * as React from 'react';
-
-import { Picker } from '@react-native-picker/picker';
-
 import type { CollectionItem, MenuPickerProps } from './MenuPicker.types';
-import { testProps } from './TestProps';
 export { CollectionItem, MenuPickerProps };
 
 /*
  * The MenuPicker was created because the RN Core Picker was deprecated (preventing us from updating to RN 0.66).
- * MenuPicker uses the community Picker package for iOS/Win32 and our own Picker for Win32/MacOS.
  */
 
-export const MenuPicker: React.FunctionComponent<MenuPickerProps> = (props: MenuPickerProps) => {
-  const { prompt, selected, onChange, collection, style, testID } = props;
-  let selectedItemKey;
-
-  collection.forEach((item) => {
-    if (item.value == selected) {
-      selectedItemKey = item.value;
-    }
-  });
-
-  return (
-    <Picker
-      prompt={prompt}
-      selectedValue={selectedItemKey}
-      onValueChange={(value: string, index: number) => onChange(value, index)}
-      style={{ ...style }}
-      /* For Android E2E testing purposes, testProps must be passed in after accessibilityLabel. */
-      {...testProps(testID)}
-    >
-      {collection.map((item, index) => (
-        <Picker.Item
-          label={item.label}
-          key={index}
-          value={item.value} /* For Android E2E testing purposes, testProps must be passed in after accessibilityLabel. */
-          {...testProps(item.testID)}
-        />
-      ))}
-    </Picker>
-  );
+export const MenuPicker: React.FunctionComponent<MenuPickerProps> = (_props: MenuPickerProps) => {
+  console.warn('Platform not supported');
```

Comment:
At this point, do we have every platform we need? I'm OK with the fallback being react-native-picker tbh

Reply:
Yeah we have a MenuPicker file for every platform. Personally for cases like this it's more intuitive to me to just have a file per platform, with the fallback just returning null (or even not having a fallback file at all if possible) - it's easier to know what platforms you're affecting rather than having to deduce which ones don't have a platform specific file. 

If we don't want to do it like this, I was also thinking we could make the win32/macOS one the fallback - both of those currently just refer to the same MenuGroup.desktop.ts file. At least the benefit of that is that iOS/Android/windows won't be bundling the MenuGroup.desktop.ts file anymore

Reply:
If it feels more intuitive to you, then we can leave as is. Thanks!